### ISSUE TITLE
Suggestion: secondary commands on target server

### ISSUE THREAD

**Issue by patchtv** [ServerSelectorX/ServerSelectorX#123]
an addon for SSX-Connector that can listen to secondary commands that SSX would send.
for example:
`menu:
  11:
    action: 'srv:smp'     # SSX sends player to smp server
    target: 'world:main' # SSX-C sends player to wold:main on target (smp) server`

**Comment by Derkades** [comment]
I agree that this would be a good feature to have, I'll look into adding it

**Comment by QuartzWarrior** [comment]
Agreed, so that people can switch servers from the target, or go back to the lobby.
